src/main/js/src/components/UI/RadioButton/RadioButton.js

```diff
@@ -1,37 +1,52 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useEffect } from 'react';
 
 import classes from './RadioButton.module.css';
 
-const radioButton = props => (
-  <div
-    className={classes.RadioButton}
-  >
-    <input
-      className={classes.RadioButtonInput}
-      name={props.name}
-      id={props.id}
-      type="radio"
-      data-cy={`radio-${props.id}`}
-      value={props.checkedValue}
-      checked={props.checkedValue === props.value}
-      onChange={props.onChange}
-      disabled={props.disabled || false}
-    />
-    <label
-      className={classes.RadioButtonLabel}
-      htmlFor={props.id}>
-      {props.label}
-    </label>
-    {props.extraLabel ? (
+const radioButton = props => {
+  useEffect(() => {
+    // Undo disabled selections in case of change
+    if (props.values &&
+      props.setFieldValue &&
+      props.name !== 'language' &&
+      props.checkedValue === props.value &&
+      props.disabled &&
+      props.values[props.name] !== '') {
```

Comment:
if condition continuation lines need better indentation

Reply:
Good point, this is fixed as well now.